### ISSUE TITLE
Is this model over-fitted?

### ISSUE THREAD

**Issue by kosonocky** [tyang816/ProtSolM#6]
I ran this model on a set of 1800 different proteins. I saved the logits / probabilities, and basically *all* of the probabilities are smashed to 0.00 and 1.00.

What's going on here?

**Comment by tyang816** [comment]
Is there any overlap between your dataset and my training dataset?

**Comment by kosonocky** [comment]
No it's all proteins that are extremely different, almost entirely under 0.3 sequence identity to anything natural.

Could it be:
- my environment? Your versioning didn't work on my end. I had to install the packages manually
- is the featurization correct?
- something else?

**Comment by kosonocky** [comment]
I guess my question is: are your logits huge in either direction, or are they more sensitive and closer to zero?

**Comment by nwod-edispu** [comment]
I have the same question, using the provided pretrained weights for prediction, the logits seem to be either 0 or 1, which does not look normal.
